Bio/Entrez/__init__.py

```diff
@@ -36,6 +36,7 @@
 
     - email        Set the Entrez email parameter (default is not set).
     - tool         Set the Entrez tool parameter (default is ``biopython``).
+    - cache        Set the Entrez cache parameter (default is not set).
```

Comment:
How about ``Set the directory to cache NCBI DTD files for the Entrez XML parser (if not set the default is under your home directory).``

We could be even more explicit about the default path?

Reply:
@peterjc Much better when it's explicit. I've replaced with your new one comment.